0-bootstrap/terraform.example.tfvars

```diff
@@ -59,6 +59,20 @@ default_region = "us-central1"
 //
 
 
+/* ----------------------------------------
+    Specific to github_bootstrap
+   ---------------------------------------- */
+// Un-comment github_bootstrap and its outputs if you want to use GitHub Actions instead of Cloud Build
+//gh_token = "YOUR-FINE-GRAINED-ACCESS-TOKEN"
```

Comment:
nit: can we add a comment here to supply this via env var rather than in plaintext?

Reply:
done